<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmOpacity</class>
 <widget class="QMainWindow" name="frmOpacity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>透明度设置</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnApply">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>应用</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbxOpacity">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>透明度(0~1)</string>
    </property>
    <widget class="QLineEdit" name="txtMaxSize">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>141</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>几何模型</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOpacityMap">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>141</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>云图</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
